website/docs/r/vapp_vm.html.markdown

```diff
@@ -97,13 +109,24 @@ The following arguments are supported:
 * `cpu_cores` - (Optional; *v2.1+*) The number of cores per socket
 * `metadata` - (Optional; *v2.2+*) Key value map of metadata to assign to this VM
 * `initscript` (Optional) A script to be run only on initial boot
-* `network_name` - (Optional) Name of the network this VM should connect to
+* `network_name` - (**Deprecated**) Name of the network this VM should connect to. **Conflicts with** with networks.
 * `vapp_network_name` - (Optional; *v2.1+*) Name of the vApp network this VM should connect to
-* `ip` - (Optional) The IP to assign to this vApp. Must be an IP address or
-  one of "dhcp", "allocated" or "none". If given the address must be within the
+* `ip` - (**Deprecated**) The IP to assign to this vApp. Must be an IP address or
+one of dhcp, allocated or none. If given the address must be within the
   `static_ip_pool` set for the network. If left blank, and the network has
   `dhcp_pool` set with at least one available IP then this will be set with
-  DHCP.
+DHCP. **Conflicts with** with networks.
+* `networks` - (Optional; *v2.2+*) List of network interfaces to attach to vm. **Conflicts with**: networks, ip. **Note**: all params of this parameter and itself do force recreation of vms!
+  * `orgnetwork` (Required) Name of the network this VM should connect to.
+  * `ip` (Optional) One of: dhcp, allocated, none or an ip.
+  * `ip_allocation_mode` (Optional) IP address allocation mode. Defaults to "POOL". One of:
+    * `POOL` (A static IP address is allocated automatically from a pool of addresses.)
+    * `DHCP` (The IP address is obtained from a DHCP service.)
+    * `MANUAL` (The IP address is assigned manually in the IpAddress element.)
+    * `NONE` (No IP addressing mode specified.)
```

Comment:
Is there still a point to keep "NONE"? One could probably omit defining NIC at all? Or am I missing something?

Reply:
I tested it and see defaulted to "DHCP" in the end. To me it looks as if this would leave source of confusion and wouldn't allow to enter such mode via validator in the new config.

Reply:
As far as I remember "NONE" wasn't working for me anyways. I think the trick would have been to not connect the network adapter if you use "NONE" but the api documentation didn't help too much either so I dropped fixing it. So we can just remove it completely.

Reply:
None is fixed and should be working properly now.